dev-java/netty-tcnative/netty-tcnative-1.1.33.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+WANT_AUTOCONF="latest"
+WANT_AUTOMAKE="latest"
+
+inherit autotools autotools-utils eutils java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Tomcat Native fork for Netty"
+
+SLOT="0"
+SRC_URI="https://github.com/netty/${PN}/archive/${PV}.zip"
+HOMEPAGE="http://netty.io/wiki/forked-tomcat-native.html"
+KEYWORDS="~amd64 ~x86"
+LICENSE="Apache-2.0"
+IUSE=""
+
+RDEPEND="dev-libs/apr:1
+	dev-libs/openssl:=
+	>=virtual/jre-1.7"
+
+DEPEND=">=virtual/jdk-1.7
+	${RDEPEND}"
+
+S=${WORKDIR}/${P}/src/main
+
+MY_S="${S}"
+
+JAVA_SRC_DIR="java"
+
+java_prepare() {
+	rm c/os_win32* || die
+
+	cd native-package || die
+
+	mv ../c src || die
+
+	SOURCES=$(ls -w 1000 src | tr "\n" " ") || die
+
+	touch Makefile.am || die
+	echo "lib_LTLIBRARIES = lib${PN}.la" > Makefile.am || die
+	echo "lib${PN/-/_}_la_SOURCES = ${SOURCES}" >> Makefile.am || die
+	echo "lib${PN/-/_}_la_LDFLAGS = -avoid-version -module -shared -export-dynamic" >> src/Makefile.am || die
```

Comment:
I went with multi-line echo which should be fine. What you are missing here, which you should KNOW is that if you have spaces preceding a variable in a Makefile.am file it will break it. So I have done it with 1 echo now vs several, but you will not like the formatting as you cannot indent the subsequent lines. Always a trade off, and readability is a moot issue really.